### ISSUE TITLE
Can't call method "chord_display" without a package or object reference

### ISSUE THREAD

**Issue by josephmturner** [ChordPro/chordpro#300]
Exporting `.cho` files to `.pdf` (by running `chordpro swing-low.cho`) yields the following error:

```
Can't call method "chord_display" without a package or object reference at /usr/local/share/perl/5.36.0/ChordPro/Output/PDF.pm line 2877.
```

## System information
 - OS: Debian 12
 - Where did you get ChordPro from? CPAN

Output of `chordpro --about`:

```
Run-time information:
  ChordPro core          6.020
  Perl                   v5.36.0 (/usr/bin/perl)
  CHORDPRO_LIB           /home/joseph/.config/chordpro
  Resource path          /home/joseph/.config/chordpro
                         /usr/local/share/perl/5.36.0/ChordPro/res

Modules and libraries:
  Storable               3.26
  Object::Pad            0.79
  Text::Layout           0.031
  File::LoadLines        1.021
  PDF::API2              2.044
  Font::TTF              1.06
```

**Comment by sciurius** [comment]
I can't reproduce this so there must be something else going on.
Can you share the song and your config?

**Comment by josephmturner** [comment]
Sorry! I had a typo in my file. There was a pair of square brackets with no chord inside: `[]`.

Perhaps you can add a more descriptive error?

**Comment by sciurius** [comment]
Thanks. FIxed in 6.020_012.
